devicetypes/smartthings/zigbee-window-shade.src/zigbee-window-shade.groovy

```diff
@@ -163,6 +164,12 @@ def open() {
 
 def setLevel(data, rate = null) {
 	log.info "setLevel()"
+	def currentLevel = device.currentValue("level")
+
+	if (isSomfy() && Math.abs(data - currentLevel) <= GLYDEA_MOVE_THRESHOLD) {
+		state.invalidSameLevelEvent = false
```

Comment:
should state.invalidSameLevelEvent be set to true in this case?

Reply:
No, by setting it to `false`, we're basically saying "hey, that next report which might has same level as now, should not be ignored"